apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -99,3 +70,43 @@ export const ScholarshipStatusRequiredStatuses = [
   'registration_sent',
   'paid'
 ];
+
+/**
+ * Valid statuses for this year's applications.
+ * Format per application type is {value: label}
+ */
+export function getApplicationStatuses(type, addAutoEmail = true) {
+  if (type === 'teacher') {
+    return {
+      unreviewed: 'Unreviewed',
+      pending: 'Pending',
+      waitlisted: `Waitlisted${autoEmailText(addAutoEmail)}`,
```

Comment:
This could be a variable calculated once at the beginning instead of calling the same function 4 times.